### PR TITLE
Refactor H2s; add scavenger-hunt

### PR DESCRIPTION
**Pull request by cutsort**
### New Content Checklist

- [ ] New content/chapter is in an Rmd file with [this kind of format and headers](https://github.com/jhudsl/OTTR_Template/blob/main/02-chapter_of_course.Rmd).

- [ ] New content/chapter contains [Learning Objectives and are in the correct format](https://github.com/jhudsl/OTTR_Template/wiki/Setting-up-images-and-graphics#learning-objectives-formatting).

- [ ] [Bookdown successfully re-renders and any new content files have been added to the _bookdown.yml](https://github.com/jhudsl/OTTR_Template/wiki/Publishing-with-Bookdown).

- [ ] Spell check runs successfully in [Github actions style-n-check](https://github.com/jhudsl/OTTR_Template/wiki/How-to-set-up-and-customize-GitHub-actions-robots#spell-check)).

- [ ] Any newly necessary packages that are needed have been added to the [Dockerfile and image](https://github.com/jhudsl/OTTR_Template/wiki/Using-Docker#adding-packages-to-the-dockerfile).

- [ ] Images are in the [correct format for rendering](https://github.com/jhudsl/OTTR_Template/wiki/Setting-up-images-and-graphics#adding-images-and-graphics-in-text).

- [ ] Every new image has [alt text and is in a Google Slide](https://github.com/jhudsl/OTTR_Template/wiki/Setting-up-images-and-graphics#accessibility).

- [ ] Each slide is described in the notes of the slide so learners relying on a screen reader can access the content. See https://lastcallmedia.com/blog/accessible-comics for more guidance on this.

- [ ] The color palette choices of the slide are contrasted in a way that is friendly to those with color vision deficiencies.
You can check this using [Color Oracle](https://colororacle.org/).

### CODE REVIEW

**Comment by KatherineCox** [comment]
@cutsort I think these are reasonable top level sections.  Just a couple of comments:

1. I'm not sure that it makes sense to put the video recording in "Preparation".  Would you watch the video if you were going to be attending a demo?  Maybe it belongs in Overview?
2. The links in the Timeline don't seem to be working in the preview.  My guess is it's just be some weirdness caused by being in the preview, since they seem to be working fine in the main branch, but it would be good to double check them after merging to make sure.

**Comment by cutsort** [comment]
> 1. I'm not sure that it makes sense to put the video recording in "Preparation".  Would you watch the video if you were going to be attending a demo?  Maybe it belongs in Overview?

The Preparation videos (which will usually be Mario renderings) should contain material necessary to introduce and complete the exercises.  They're for asynchronous use and would be delivered live during synchronous events.  Is it better now that it's streamlined to "Watch Video" with the slides as a reference link?

> 2. The links in the Timeline don't seem to be working in the preview.  My guess is it's just be some weirdness caused by being in the preview, since they seem to be working fine in the main branch, but it would be good to double check them after merging to make sure.

Links appear to work when rendered locally so perhaps preview weirdness like you said.

**Comment by KatherineCox** [comment]
> The Preparation videos (which will usually be Mario renderings) should contain material necessary to introduce and complete the exercises. They're for asynchronous use and would be delivered live during synchronous events. Is it better now that it's streamlined to "Watch Video" with the slides as a reference link?

Maybe make that explicit then?  I think it would be helpful to make it clear which "Preparation" activities are needed by everybody (e.g. cloning the Workspace) vs. what's intended only for asynchronous use.

File: 01-what-is-anvil.Rmd [hunk 1]
```diff
@@ -53,19 +101,3 @@ _Programming skills_
...
-
-<br>
-
-```{r, echo=FALSE, out.width="100%", fig.alt="Title page of a series of slides for 'Welcome to AnVIL'."}
-ottrpal::include_slide("https://docs.google.com/presentation/d/1wTcGBM-ZOo7SmyR7x_BCvau2bBig01Er59s4lXb1Bg4/edit#slide=id.p")
-```
-
-# Demo Recording
-
-Please check out our recording of this demo here:
```
Comment (KatherineCox) [inline 01-what-is-anvil.Rmd:68]: I'm confused from this sentence about what the "Demo" is - does it include the Exercises or not?  This says it's a recording of the demo, but it's just  the "Welcome to AnVIL" presentation with no scavenger hunt.

Also "recording of this demo" implies to me that I would watch this video if I wanted to get a quick, passive look at the materials, i.e. I don't want to put in the time/effort to work through them myself right now, just get an idea of what's there.  If I intended to work through the materials, I'd probably skip this video if I thought it was just a recording of the stuff I'm about to do myself.

Comment (cutsort) [reply 01-what-is-anvil.Rmd:68]: "Demo Recording" has been replaced by "Watch Video" as it's only a presentation of the slidedeck and not a recording of the exercises.

**Comment by cutsort** [comment]
> Maybe make that explicit then? I think it would be helpful to make it clear which "Preparation" activities are needed by everybody (e.g. cloning the Workspace) vs. what's intended only for asynchronous use.

How much of this repo do you think learners will be using during a synchronous event (vs being assigned specific sections as prep work, as reference after the event, completely asynchronous)?

File: 01-what-is-anvil.Rmd (hunk 1 above)
Comment (KatherineCox) [reply 01-what-is-anvil.Rmd:68]: Oh right, sorry, was a bit distracted while reviewing.

**Comment by KatherineCox** [comment]
> How much of this repo do you think learners will be using during a synchronous event (vs being assigned specific sections as prep work, as reference after the event, completely asynchronous)?

### TLDR

  - Can we make it work for all those cases?  If no (or not well), who's the primary audience for the book?  Do we want to prioritize asynchronous learners?
  - Can we at least clarify the purpose of the video for instructors / authors?  *I'm* confused about what the purpose of the video is.  Maybe that clarification belongs in the instructor guide? and the Preparation section is fine as is?

### Longer ramblings:

#### Who's the audience?

I think you're right, a lot of the confusion here is coming from lack of clarity about the audience(s) for the book.  Instructors? Synchronous learners?  Asynchronous learners?  I think it's possible to mostly harmonize things so that it works reasonably well for everybody, with maybe a couple awkward rough edges.

In places where it's awkward, then does it make sense to prioritize asynchronous learners > instructors > synchronous learners?  Since asynchronous learners have only the book to rely on?  And synchronous learners rely on instructors?

#### Thoughts on when/how the repo would be used in the context of a live event:

- Where possible, lean towards flexibility.  Can we set it up so that an instructor can choose whether they want to link students to the book for their particular event?
- I haven't actually run an AnVIL event live.  How much trouble is it getting people to follow along (e.g. cloning)? Do they usually have any information available other than the slides?  Would it be helpful if they did?
- As a participant, I really like it when the materials are available for me to jump back and forth a bit during a synchronous event.  That could be slides, or could be the OTTR book, I don't have a strong preference there.
- If I were teaching, I'd lean towards linking people to the book (assuming it's reasonably faithful to what I intend to do live), even if that gives people freedom to get out-of-sync with me.  My thought process there is basically "My audience are adults, they have their own ideas about what they want to get out of the demo, if they want to pay full attention and stay in sync with me, great; if they want to work ahead if I'm too slow for them, or fall behind because they went off Googling something and then rely on the book instead of following me, that's fine too."  That's just my personal inclination.

#### Thoughts/questions on the purpose of the video:

- Is the Welcome to AnVIL demo unusual for including so much "talking" up front?  Are most demos likely to have a significant (say, >3 min) presentation component that would fit under "Preparation"?
- Related - what "material necessary to introduce and complete the exercises" is best as a video vs. prose?
- My personal inclination there is to use a video for a short, high level overview / motivation, and I think that's leading to my suggestion about putting the video in the Overview rather than Preparation, since I'm thinking of it more as "here's the big picture".  It sounds like you're thinking of it more as delivering some necessary information people need to get started, in which case I agree it makes sense to put it in Preparation.  So...stepping back what's the goal of the video (and is Welcome to AnVIL special in that regard)?


### Looking back at the chapter with those thoughts in mind

- If, (a) for the book, we prioritize asynchronous learners and (b) the video contains content they *need* to watch before starting, then
  - I think what you have is good for the Preparation chapter - makes sense for an asynchronous learner, and in a live setting the instructor can tell people to skip the video
  - I'd add something in the instructor guide that clarifies what the video is for and when/whether they would want to assign the video vs. covering the material themselves
- Otherwise
  - If we don't think we'll have a lot of asynchronous learners and therefore don't want to prioritize them, then maybe it makes sense to explicitly comment that the video is redundant with the live presentation.
  - If the videos are typically more of a quick overview / big picture / motivator, consider putting them in Overview.  (Maybe making an exception for Welcome to AnVIL?)

I think most of my suggestions about moving the video / changing the messaging are coming from different assumptions about those two points (which I couldn't have told you before writing this essay, haha).  It would be good to have some clarity there.

**Comment by cutsort** [comment]
> * Is the Welcome to AnVIL demo unusual for including so much "talking" up front?  Are most demos likely to have a significant (say, >3 min) presentation component that would fit under "Preparation"?

Yes, what-is-anvil with its 20 min slidedeck and 20 min exercises differs from the scale-with-workflows and the bioconductor/galaxy demos with their 5 min slidedeck and 15 min exercises for the NHGRI Intramural workshop (30-min for monthly AnVIL Demos series, 45-min for asynchronous).
